packages/manager/modules/anthos/src/dashboard/general-information/order-host/order-host.html

```diff
@@ -0,0 +1,11 @@
+<oui-modal
```

Comment:
Please re-use component from https://github.com/ovh/manager/pull/5425/files#diff-0dd4835c8810f3cc55560b975ec752228475da61f0fb531bf8c97871e5a2fe4f

Reply:
@ganeshkumar1989, to avoid blocking review, we can plan to reuse the component an another task.

Reply:
Please create a task and reference the task id here, in order to track it please

Reply:
MANAGER-7520.
Done